app/mesh/changelog.md

```diff
@@ -6,18 +6,29 @@ no_version: true
 
 ## 1.6.0
 
-> Released on TBA
+> Released on 2022/02/24
 
 ### Changes
 
-Built on top of [Kuma <version>](changelog link)
+Built on top of [Kuma 1.5.0](https://github.com/kumahq/kuma/releases/tag/1.5.0)
 
-- features TBA
+- UBI images support.
+- ECS EC2 and Fargate first party support.
+- Update OPA agent to v0.37.2.
 
 ### Upgrading
 
-Info TBA
+- The `kuma.metrics.dataplane.enabled` and `kuma.metrics.zone.enabled` configurations have been removed. Kuma always generate the corresponding metrics.
+- We removed support for old Ingress (`Dataplane#networking.ingress`) from pre 1.3 days. If you are still using it, please migrate to ZoneIngress first (see [Kuma Upgrade to 1.2.0 section](https://github.com/kumahq/kuma/blob/master/UPGRADE.md#upgrade-to-120)).
 
+#### Kubernetes
+- Please migrate your kuma.io/sidecar-injection annotations to labels. The new version still supports annotation, but to have a guarantee that applications can only start with sidecar, you must use label instead of annotation.
```

Comment:
```suggestion
- Migrate your kuma.io/sidecar-injection annotations to labels. The new version still supports annotations, but to  guarantee that applications can only start with a sidecar, you must use a label instead of an annotation.
```